dbms/programs/performance-test/PerformanceTestInfo.cpp

```diff
@@ -1,6 +1,6 @@
 #include "PerformanceTestInfo.h"
 #include <Common/getMultipleKeysFromConfig.h>
-#include <Common/SettingsChanges.h>
+#include <Common/SettingsChanges.h> ///
```

Comment:
Empty commit is better way to trigger CI.

Reply:
`--amend` with simple changes to commit messages is another way. (empty commits might lead to weird git history)